test/unit/selector.js

```diff
@@ -109,7 +109,7 @@ QUnit.test( "child and adjacent", function( assert ) {
 	assert.t( "Child", "p>a", [ "simon1","google","groups","mark","yahoo","simon" ] );
 	assert.t( "Child w/ Class", "p > a.blog", [ "mark","simon" ] );
 	assert.t( "All Children", "code > *", [ "anchor1","anchor2" ] );
-	assert.t( "All Grandchildren", "p > * > *", [ "anchor1","anchor2" ] );
+	assert.u( "All Grandchildren", "p > * > *", [ "anchor1","anchor2" ] );
```

Comment:
I agree with @gibson042. Changing `assert.t` will require lots of changes so it'd be better done separately to not add too much work to this PR.